application/templates/finished.html

```diff
@@ -0,0 +1,42 @@
+{% extends "base/lr_base_layout.html" %}
+
+{% block content %}
+
+  <div class="main" id="content">
+    {{ super() }}
+
+    <div class="inner-block">
+      <div class="grid-row">
+        <div class="column-two-thirds">
+
+          <div class="govuk-box-highlight">
+            <p></p>
+
+            <h1 class="bold-large">Thank you for viewing your</br> mortgage deed</h1></p>
```

Comment:
After discussions with designers it was agreed to use a </br> tag here to wrap the text as desired as the message is only temporary until the signing service is implimented